### ISSUE TITLE
[Dialogs] [FeatureHighlight] When using an modal in VO mode, dismiss should select the original sender

### ISSUE THREAD

**Issue by ianegordon** [material-components/material-components-ios#1307]
If you present an modal, such as an MDCAlertController, in VO mode, then dismiss it, the first accessible element on the interface is selected.

Should select the element that was previously selected before presentation?

This also applies to Feature Highlight.

---
#### Internal data
- Associated internal bug: [b/117179467](http://b/117179467)

**Comment by romoore** [comment]
Closing this due to inactivity greater than 6 months. Please feel free to reopen if you believe it should be worked on in the near future.
